MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/AddNewEntryManually.java

```diff
@@ -135,15 +148,39 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
         }
 
         if((EditorInfo.IME_ACTION_DONE == actionId) && (v.getId() == R.id.et_add_cover)) {
-            mAddCover = addCover.getText().toString();
-            Log.v(TAG, mAddCover);
-            coverPreview.setVisibility(View.VISIBLE);
-            Picasso.with(this)
-                    .load(mAddCover)
-                    .into(coverPreview);
+
+            if (addCover.getText().length() == 0) {
```

Comment:
Same comment as above. And now we notice a little bit of duplication, in these cases it is advisable to extract a method and use it in both places so we have just one point of change. We can extract the following method:

```java
private void loadCover() {
    if (addCover.getText().length() > 0) {
        mAddCover = addCover.getText().toString();
    }

    coverPreview.setVisibility(View.VISIBLE);
    Picasso.with(this)
                .load(mAddCover)
                .into(coverPreview);
}
```

and use it here and above.